be/src/services/vocabulary.service.ts

```diff
@@ -116,10 +116,11 @@ export class VocabularyService {
     const { options, filter } = this.transformToMongoDBFilterOption(query);
     // @ts-ignore
     const dbos: Vocabulary[] = (await VocabularyService.collection().find(filter, options).toArray()) as Vocabulary[];
+    const size = dbos.length
     return {
       offset: query.offset,
       rows: dbos.length,
-      totalItems: 0, // TODO
+      totalItems: size, // TODO
```

Comment:
1. Remove the `TODO`comment
2. From my point of view, using the variable with the same content as the row above is wrong. Instead:
3. We need here the total items of the filter + options without any paging applied